CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Remove camelCase names on `CREATE ANALYZER`.
```

Comment:
pls mark as breaking.

i think the whole change is that the names are case insensitive? however i am not sure. i think we should treat it as an identifier, is this possible?